src/test/java/DemoTest.java

```diff
@@ -1,7 +1,125 @@
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+import java.util.Random;
+import java.util.concurrent.TimeUnit;
 
 public class DemoTest {
 
+  @Test
+  public void incorrectPassAndEmailTest() {
+    WebDriver driver = new ChromeDriver();
+    driver.manage().timeouts().implicitlyWait(10, TimeUnit.SECONDS);
+    driver.get("https://deens-master.now.sh");
+
+    driver.findElement(By.cssSelector("a[href*='login']")).click();
+
+    WebElement email = driver.findElement(By.cssSelector("#email"));
```

Comment:
you use the same locators in each test. It would be better to create static variable and store locator there and use it in different places